doc/userguide/src/SupportingTools/Libdoc.rst

```diff
@@ -190,7 +197,7 @@ are omitted if getting them from the library fails for whatever reason.
 
 Libdoc automatically uses the XML format if the output file extension is
 :file:`*.xml` or :file:`*.libspec`. When using the special :file:`*.libspec`
-extension, Libdoc automatically enables the `xml:html` format which means
+extension, Libdoc automatically enables the options `-f XML -s HTML` which means
```

Comment:
Please use long option formats in examples like this to make them easier to understand and to be consistent with examples elsewhere in the UG.